src/test/java/io/vertx/core/net/ConnectionPoolTest.java

```diff
@@ -487,6 +487,26 @@ public void testDiscardExpiredConnections() {
     // assertWaitUntil(mgr::closed);
   }
 
+  @Test
```

Comment:
why is this test necessary ? is it a new test to ensure a behavior that was not tested previously ?

Reply:
It's actually not necessary. I got confused before. I am removing it.